modules/upgrade-upgrading-cli.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrading-cluster.adoc
+
+[id='upgrade-upgrading-web-{context}']
```

Comment:
@jiajliu, after the default update server works the way it's supposed to, will the CLI process look like this?

Reply:
@kalexand-rh 
Could i ask if this doc is targeted for customer? or just a internal version? Cause I noticed it mostly came from doc[1] wrote to help QE do pre-release test(non-production). I'm not sure if the steps in the doc should be an official/supported way for customer doing update. So if this "upgrade draft" doc is for customer, it's better to ask dev to give an official document about it.

Such as following part, afaik, for production cluster, upstream is not needed to be changed and this internal fake upstream should not be show in the doc.
```
$ curl https://openshift-release.svc.ci.openshift.org/graph
```
And if it's for clustomer, I think following make users confused.
> For non-production clusters, you can modify the specification that describes the cluster version state that you want:

This is incomplete for customer to check if the cluster is updated or not. It should include many componants here. But if just have a basic check, maybe run "oc get clusterversion" is enough.
> After the update completes, you can confirm that the Cluster Version Operator is using the new image:

[1] https://docs.google.com/document/d/1CfsJ3wSOtHZybKFV2CF-7t3tZiEsFFjP5bCztkXb94U/edit#